data-manipulation/encryption/create-new-key-via-cryptacquirecontext.yml

```
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: create new key via CryptAcquireContext
+    namespace: data-manipulation/encryption
+    author: chuong.dong@fireeye.com
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+    mbc:
+      - Cryptography::Encryption Key [C0028]
+    references:
+      - https://docs.microsoft.com/en-us/windows/win32/api/wincrypt/nf-wincrypt-cryptacquirecontexta
+    examples:
+      - E5369AC309F1BE6D77AFEEB3EDAB0ED8:0x401350
+      - 849945BB1D222607446C7A917F0D935A:0x405482
+  features:
+    - and:
+      - or:
+        - api: crypt32.CryptAcquireContextA
+        - api: crypt32.CryptAcquireContextW
```

```suggestion
        - api: crypt32.CryptAcquireContext
```

capa will automatically search for A/W as appropriate